README.md

```diff
@@ -1,10 +1,53 @@
-# Problems in the Python C API
+# C API Workgroup
 
-This workgroup was set up following the [recent Language Summit discussions about the C API](https://pyfound.blogspot.com/2023/05/the-python-language-summit-2023-three.html).
+The workgroup idea was initiated following the [recent Language Summit discussions about the C API](https://pyfound.blogspot.com/2023/05/the-python-language-summit-2023-three.html).
 
-The purpose here is to identify the problems in the current C API, and to determine which of them can be fixed incrementally.
-Please create an issue in the repo for each problem you identify, and that particular issue can be discussed there.
+At this point, we don't have a formal setup for the workgroup yet, but in order to start, we are collecting input from the community to hash out where we'd like to go with the Python C API.
+
+The formal setup of the workgroup will then be a second step.
+
+## Goals
+
+The overarching goal of the workgroup is to create and maintain a design for a consistent, complete and useful Python C API.
```

Comment:
> I very much like the idea of having a workgroup for the task,

We need to first agree what the task is. In this PR you are editing the README of a repo called _problems_, and making it about _solutions_.

Reply:
That is true to some extent, since a *workgroup* would help to eventually solve the problems, but we have to start somewhere, right ?

Perhaps we ought to have a separate repo called "workgroup" or "charter" (that's how these more formal things are called in the PSF workgroups) and add the part about the workgroup there.

Reply:
We left the Language Summit with a clear task, to produce a list of problems. In Brno let’s set a new task and form a workgroup to do it. Until then we can only clarify what *this* repo is for.